rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/LocalJobFriendlyPackerCommandFactory.groovy

```diff
@@ -26,7 +26,18 @@ class LocalJobFriendlyPackerCommandFactory implements PackerCommandFactory {
     def packerCommand = [baseCommand, "packer", "build", "-color=false"]
     parameterMap.each { key, value ->
       if (key && value) {
-        def keyValuePair = value instanceof String && value.contains(" ") ? "$key=\"$value\"" : "$key=$value"
+        def cleansedValue = value instanceof String ? value.trim() : value
```

Comment:
Removal of the extra leading and trailing quotes are needed as when passing the "packages" parameter to packer. I'm very new to Groovy, but my understanding is that trim() doesn't do that.

The command line that was being generated by Spinnaker was `packer build ... -var packages="googlechrome javaruntime" azure-windows-2012-r2.json`. If you ran this same command line via an interactive shell and then looked at /proc/<proces_id>/cmdline, you'll see it as:

```
ubuntu@spinnaker:~$ cat /proc/7472/cmdline
/usr/bin/packerbuild-color=false-debug-varazure_client_id=##########ClientID##################-varazure_client_secret=#########-varazure_resource_group=spinnaker-varazure_storage_account=spinnakerimages-varazure_subscription_id=########SubscriptionID##############-varazure_tenant_id=##########TenantID##################-varazure_object_id=##########ObjectID##################-varazure_location=eastus-varazure_image_publisher=MicrosoftWindowsServer-varazure_image_offer=WindowsServer-varazure_image_sku=2012-R2-Datacenter-varazure_image_name=null-null-varpackage_type=nupkg-varpackages=googlechrome javaruntime-varconfigDir=/opt/rosco/config/packer/opt/rosco/config/packer/azure-windows-2012-r2.json
```

This looks fine as you don't see the quotes. When Spinnaker runs packer via a non-interactive shell and you look at /proc/<proces_id>/cmdline, you'll see it as:

```
ubuntu@spinnaker:~$ cat /proc/7619/cmdline
/usr/bin/packerbuild-color=false-debug-varazure_client_id=##########ClientID##################-varazure_client_secret=#########-varazure_resource_group=spinnaker-varazure_storage_account=spinnakerimages-varazure_subscription_id=########SubscriptionID##############-varazure_tenant_id=##########TenantID##################-varazure_object_id=##########ObjectID##################-varazure_location=eastus-varazure_image_publisher=MicrosoftWindowsServer-varazure_image_offer=WindowsServer-varazure_image_sku=2012-R2-Datacenter-varazure_image_name=null-null-varpackage_type=nupkg-varpackages="googlechrome javaruntime"-varconfigDir=/opt/rosco/config/packer/opt/rosco/config/packer/azure-windows-2012-r2.json
```

You'll notice the extra quotes for the 'packages' variable, which causes packer to pass the parameter as `packages=""googlechrome javaruntime""` to Powershell which causes a parsing error.